### PR TITLE
Add poster only template for media atoms

### PR DESCRIPTION
**Pull request by gidsg**
## What does this change?
Adds a template for case where media atom has `posterUrl` but no assets. This is because editorial may wish to preview or publish an media atom before assets are available.

## What is the value of this and can you measure success?
Fix previous exception when a media atom with no assets was embedded. Support poster only case. Adds caption icon across media atom cases.

## Does this affect other platforms - Amp, Apps, etc?
Adds AMP template for poster only

## Screenshots
.com

screen shot 2016-12-15 at 10 36 38

AMP

screen shot 2016-12-15 at 10 36 23

### CODE REVIEW

File: common/app/views/fragments/atoms/mediaPosterOnly.scala.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@(media: model.content.MediaAtom)
+<figure
+itemprop="associatedMedia image"
+itemscope
+itemtype="http://schema.org/ImageObject"
+data-component="image" xmlns="http://www.w3.org/1999/html"
+class="element element-image">
+    <div class="u-responsive-ratio">
+            <img
```
Comment (gidsg) [inline common/app/views/fragments/atoms/mediaPosterOnly.scala.html]: I am just using a simple `` tag here as we only have a single image URL.

Comment (akash1810) [inline common/app/views/fragments/atoms/mediaPosterOnly.scala.html]: there is a list of poster image assets on the atom that can be used

Comment (akash1810) [inline common/app/views/fragments/atoms/mediaPosterOnly.scala.html]: is this passed through imgix?

**Review by akash1810 — APPROVED** [review]
👍 after rebasing to get build to pass

File: common/app/views/fragments/atoms/mediaPosterOnly.scala.html (hunk 1 above)
Comment (gidsg) [inline common/app/views/fragments/atoms/mediaPosterOnly.scala.html]: Yes, it is passed through Imgix in the building of the case class

Comment (gidsg) [inline common/app/views/fragments/atoms/mediaPosterOnly.scala.html]: Thanks for letting me know about the `PosterImage` changes, I would rather merge this using the old CAPI model and pick up that work as a separate PR.

**Pull request merged**